odoo/addons/base/views/res_partner_views.xml

```diff
@@ -145,10 +145,10 @@
             <field name="arch" type="xml">
                 <form string="Partners">
                 <div class="alert alert-warning oe_edit_only" role="alert" name="warning_tax" invisible="not same_vat_partner_id">
-                  A partner with the same <span><span class="o_vat_label">Tax ID</span></span> already exists (<field name="same_vat_partner_id" context="{'show_address': False, 'show_vat': False}"/>), are you sure you want to create a new one?
+                  A partner with the same <span><field name="vat_label" /></span> already exists (<field name="same_vat_partner_id" context="{'show_address': False, 'show_vat': False}"/>), are you sure you want to create a new one?
```

Comment:
Note that this is not great for translators because they lose the context of the sentence (which is now split, see the changes in the `.pot`)

Not sure if we can improve that :thinking:

Reply:
I got your point but I didn't find any other ways to add that to `.pot` 
Actually, I went with the safest way of exporting the `.pot` file and got the changes I needed from it(which is to split it like in the `.pot` changes)

Reply:
It's not something to do with the translation files themselves, but with the xml code and how it can be split in the translation files.

Reply:
hmmm, I'm not sure if there's another way to write it

Reply:
We could have post processed it in the `_get_view`, but after discussion with i18n team, it is probably overkill
https://discord.com/channels/678381219515465750/1171438356840857620/1318193745761402941